pyproject.toml

```diff
@@ -4,3 +4,23 @@ requires = [
     "wheel >= 0.30.0",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.black]
+line-length = 99
+target-version = ['py36']
+extend-exclude = '''
```

Comment:
Can we add `skip-magic-trailing-comma = true`?

Reply:
Why's that? Is it sus?

Reply:
Yeah, it's sus, especially for an initial reformat. It results in less compact code.